src/components/admin/technical-maintenance-tab.yml

```diff
@@ -0,0 +1,33 @@
+tabInfo:
+  en: You can trigger different tasks from here, in order to execute technical maintenance over the database and the CDN.
+  de: You can trigger different tasks from here, in order to execute technical maintenance over the database and the CDN.
+documentsValidationHeader:
+  en: Documents validation
+  de: Dokumente Validierung
+documentsValidationDescription:
+  en: Verify that all document revisions and documents in the database contain valid data. This task does not change any database data.
+  de: Verify that all document revisions and documents in the database contain valid data. This task does not change any database data.
+documentsRegenerationHeader:
+  en: Documents regeneration
+  de: Dokumente Neuerstellung
+documentsRegenerationDescription:
+  en: Re-create all documents based on their document revisions. This tasks updates database data, possibly resulting in data corrections.
+  de: Re-create all documents based on their document revisions. This tasks updates database data, possibly resulting in data corrections.
+cdnResourcesConsolidationHeader:
+  en: CDN resources consolidation
+  de: CDN-Ressourcen Konsolidierung
+cdnResourcesConsolidationDescription:
+  en: Update the cdnResources array on documents and documentRevisions. This task updates database data, possibly resulting in data corrections.
+  de: Update the cdnResources array on documents and documentRevisions. This task updates database data, possibly resulting in data corrections.
```

Comment:
Dieser Task aktualisiert die Liste der in den Dokumenten und Dokument-Revisionen verknüpften CDN-Ressourcen. Eventuell inkorrekte Daten werden dabei berichtigt.